ferrocene/doc/qualification-plan/src/standards/iec-61508.rst

```diff
@@ -12,167 +12,209 @@ The following are sections in |iec_ref| Part 3.
    * - Clause
      - ``rustc``
      - ``rustfmt``
+     - ``ferrocene-self-test``
 
    * - 7.4.4.1
      - N/A
      - N/A
+     - N/A
 
    * - 7.4.4.2
      - N/A
      - N/A
+     - N/A
 
    * - 7.4.4.3
      - This qualification is the justification
-     - This qualification is the justification
+     - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 7.4.4.4
      - | :doc:`Safety Manual - rustc - Usage <safety-manual:rustc/usage>`
        | :doc:`Safety Manual - rustc - Degraded environment <safety-manual:rustc/degraded-environment>`
        | :doc:`Evaluation Report - rustc - Use cases <evaluation-report:rustc/use-cases>`
      - | :doc:`Safety Manual - rustfmt - Usage <safety-manual:rustfmt/usage>`
        | :doc:`Evaluation Report - rustfmt - Use cases <evaluation-report:rustfmt/use-cases>`
+     - | :doc:`User Manual - ferrocene-self-test - Usage <user-manual:self-test/using>`
+       | :doc:`Evaluation Report - ferrocene-self-test - Use cases <evaluation-report:self-test/use-cases>`
 
    * - 7.4.4.5
      - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
      - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Tool analysis <evaluation-report:self-test/tool-analysis>`
 
    * - 7.4.4.6
      - | :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
        | :doc:`Qualification Report - rustc - Test results <qualification-report:rustc/index>`
-     - :ref:`Evaluation Report - rustfmt - Tool analysis - IEC 61508 Tool Classification <rustfmt_iec_tool_classification>`
+     - :ref:`Evaluation Report - rustfmt - Tool analysis - Tool Classification <rustfmt_iec_tool_classification>`
+     - :ref:`Evaluation Report - ferrocene-self-test - Tool analysis - Tool Classification <ferrocene_self_test_iec_tool_classification>`
 
    * - 7.4.4.7
      - See sub items below
      - See sub items below
+     - See sub items below
 
    * - 7.4.4.7.a
      - :doc:`Qualification Plan - Validation process <qualification-plan:validation>`
      - N/A
+     - :doc:`Qualification Plan - Validation process <qualification-plan:validation>`
 
    * - 7.4.4.7.b
      - :doc:`Document List <document-list:index>`
-     - N/A
+     - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 7.4.4.7.c
      - :doc:`Evaluation Report - Use cases <evaluation-report:rustc/use-cases>`
      - N/A
+     - :doc:`Evaluation Report - Use cases <evaluation-report:self-test/use-cases>`
```

Comment:
I deliberately removed because it was becoming so long that it required a horizontal scroll. Was even thinking that we should remove these tool names in the rows, for they are implied by the column names.

Reply:
Ok